docs/cloud/moving-into.md

```diff
@@ -0,0 +1,55 @@
+## When to migrate to K8s
+
+When considering migrating your services from Pouta to k8s, it’s important to consider the relative
+strengths of each platform. Both Pouta and Kubernetes-based platforms, such as Rahti, are Cloud
+Computing platforms, but there are some important differences. Virtual machine based services, like
+cPouta, are classified as Infrastructure-as-a-Service (IAAS) cloud services. Kubernetes on the other
```

Comment:
Do we have any article to link for IAAS and PAAS further study? Internal or external like the wikipedia?